File: PHP-COPY-JS/12.php
<?php

// Task 1
function twoSum($array1, $array2) {
    $somaUm = "";
    $somaDois = "";

    foreach ($array1 as $element) {
        $somaUm .= $element;
    }

    foreach ($array2 as $element) {
        $somaDois .= $element;
    }

    $somaTotal = intval($somaUm) + intval($somaDois);
    echo $somaTotal . "\n";
    return $somaTotal;
}

twoSum([1, 2, 3], [0, 7]);

// Task 2
function luckyNumber($value) {
    $valorOriginalString = strval($value);
    $valorInvertido = strrev($valorOriginalString);

    echo $valorInvertido . "\n";
    echo $valorOriginalString . "\n";

    if ($valorOriginalString === $valorInvertido) {
        echo "True\n";
        return true;
    } else {
        echo "False\n";
        return false;
    }
}

luckyNumber(524425);

// Task 3
function errorMessage($input) {
    if ($input === null || $input === "" || trim(strval($input)) === "") {
        echo "Required field\n";
        return "Required field";
    }

    // Tenta converter a entrada para número
    $parsedNumber = floatval($input);

    // Verifica se a entrada não é um número válido ou é zero
    if (is_nan($parsedNumber) || $parsedNumber === 0) {
        echo "Must be a number besides 0\n";
        return "Must be a number besides 0";
    }

    // Se passar todas as verificações, retorna uma string vazia
    echo "Empty\n";
    return "";
}

errorMessage(null);
?>

File: PYTHON3-COPY-JS/12.py
def two_sum(array1, array2):
    soma_um = ""
    soma_dois = ""

    for element in array1:
        soma_um += str(element)

    for element in array2:
        soma_dois += str(element)

    soma_total = int(soma_um) + int(soma_dois)
    print(soma_total)
    return soma_total

two_sum([1, 2, 3], [0, 7])

# Task 2

def lucky_number(value):
    valor_original_string = str(value)
    valor_invertido = valor_original_string[::-1]

    print(valor_invertido)
    print(valor_original_string)

    if valor_original_string == valor_invertido:
        print("True")
        return True
    else:
        print("False")
        return False

lucky_number(524425)

# Task 3

def error_message(input):
    if input is None or input == "" or str(input).strip() == "":
        print("Required field")
        return "Required field"

    try:
        parsed_number = float(input)
        if parsed_number == 0:
            print("Must be a number besides 0")
            return "Must be a number besides 0"
    except ValueError:
        print("Must be a number")
        return "Must be a number"

    print("Empty")
    return ""

error_message(None)
